readme.md

```diff
@@ -433,7 +499,6 @@ You can configure the Nextcloud container to use a specific directory on your ho
 
 - An example for Linux is `-e NEXTCLOUD_DATADIR="/mnt/ncdata"`.
 - On macOS it might be `-e NEXTCLOUD_DATADIR="/var/nextcloud-data"`
-- For Synology it may be `-e NEXTCLOUD_DATADIR="/volume1/docker/nextcloud/data"`. 
```

Comment:
Lets keep this

Reply:
done with https://github.com/nextcloud/all-in-one/pull/1813/commits/fb7f61daff23ef41c5c9cdb9d5ce5f41fce6ec4a